README.md

```diff
@@ -85,6 +85,13 @@ vbatts@valse ~/src/vb/git-validation (master) $ echo $?
 1
 ```
 
+Excluding paths that are out of the scope of your project:
+```bash
```

Comment:
Don't ask me [how I know this][1] ;).

[1]: https://github.com/wking/oci-command-line-api/commit/770453114b748d4746ba1614a1e9a5d9003803fe